neuralforecast/models/timellm.py

```diff
@@ -319,9 +327,7 @@ def __init__(
         n_heads: int = 8,
         enc_in: int = 7,
         dec_in: int = 7,
-        llm=None,
-        llm_config=None,
-        llm_tokenizer=None,
+        llm_model: str = "GPT2",
```

Comment:
Same, ideally we don't deprecate existing arguments

Reply:
I’ve updated it to reuse the existing variable names as you suggested.